src/laravel/pagseguro/Sender/Sender.php

```diff
@@ -142,11 +142,11 @@ public function setName($name)
      */
     public function setPhone($phone)
     {
-        if ($phone === null) {
-            $phone = [];
+        if (!is_null($phone)) {
+            $this->phone = Phone::factory($phone);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\laravel\pagseguro\Phone\Phone' in method 'setPhone'.](https://app.codacy.com/manual/michaeldouglas010790/laravel-pagseguro/pullRequest?prid=5029546)